service/create/service.go

```diff
@@ -50,6 +61,10 @@ func New(config Config) (*Service, error) {
 		return nil, microerror.MaskAnyf(invalidConfigError, "logger must not be empty")
 	}
 
```

Comment:
Cosmetic nitpicking. The empty line here is not necessary. We can simply have the conditionals all together unless there is a deeper reason behind grouping them.